CommandsManual.md

```diff
@@ -274,36 +326,109 @@ Each Project file contains settings for a Project, an *impt* entity which links
 
 A Project file may affect commands called from the directory where the file is located. Product, Device Group, Devices, Deployment, and source code files referenced by Project file may be assumed by a command when they are not specified explicitly.
 
+**Note** Currently, Builder information in the project file must be created manually.
+
+Each project file contains the following:
+
+- Global Builder information: includes `variables` which are used by all device groups, and `libs` which are `.js` files that contain some functions used by builder variables
```

Comment:
I think `preBuild` and `postBuild` can also be included in the Global builder section.

Reply:
I believe `preBuild` and `postBuild` are available in the Global builder info.

And, with that in mind, what happens if `preBuild` and `postBuild` are included in the Global and Local sections of the build file? What is the order of execution? That should be specified (and it should probably be:

Global PreBuild
Local PreBuild
Local PostBuild
Global PostBuild

Reply:
I'm sure that `preBuild` execute before `postBuild`  not matter what. However, I'm not sure as to whether the global scripts would execute before the local within `preBuild` or `postBuild` it depends on how `deepmerge` works